ios/NotificationsHelperIOS10.swift

```diff
@@ -68,8 +70,15 @@ class NotificationsHelperIOS10: NSObject, UNUserNotificationCenterDelegate {
     let category = UNNotificationCategory(identifier: NotificationsHelperIOS10.CATEGORY, actions: [action], intentIdentifiers: [], options: [])
     UNUserNotificationCenter.currentNotificationCenter().setNotificationCategories([category])
     
-    let request = UNNotificationRequest(identifier: url, content: content, trigger: nil)
-    UNUserNotificationCenter.currentNotificationCenter().addNotificationRequest(request, withCompletionHandler: nil)
+    let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 1, repeats: false)
```

Comment:
You can trigger notifications with some delays or at a specific date.

Will comment on single line, thought it was pretty readable ... oh gosh, maybe i'm getting used to swift .... is that ok?